browser-interface/scripts/build.ts

```diff
@@ -0,0 +1,136 @@
+import * as glob from "glob"
+import { writeFileSync, readFileSync } from "fs"
+import * as path from "path"
+import { copyFile, ensureFileExists } from "./utils"
+
+import { OutputOptions, rollup } from "rollup"
+import typescript from "rollup-plugin-typescript2"
+import resolve from "rollup-plugin-node-resolve"
+import commonjs from "rollup-plugin-commonjs"
+// import globals from "rollup-plugin-node-globals"
+
+const PROD = !!process.env.CI
+
+console.log(`production: ${PROD}`)
+
+const plugins = [
+  typescript({
+    verbosity: 2,
+    clean: true,
+    useTsconfigDeclarationDir: true,
+  }),
+  resolve({
+    browser: true,
+    preferBuiltins: false,
+  }),
+  commonjs({
+    ignoreGlobal: true,
+    include: [/node_modules/],
+    namedExports: {},
+  }),
+
+  // globals({}),
+]
+
+process.env.BUILD_PATH = path.resolve(
+  process.env.BUILD_PATH || path.resolve(__dirname, "../../unity-renderer/Builds/unity")
+)
+const DIST_PATH = path.resolve(__dirname, "../dist")
+
+async function main() {
+  await copyBuiltFiles()
+  await buildRollup()
+  await createTypings()
+  await createPackageJson()
+}
+
+async function copyBuiltFiles() {
+  const basePath = path.resolve(process.env.BUILD_PATH!, "Build")
+
+  for (let file of glob.sync("**/*", { cwd: basePath, absolute: true })) {
+    copyFile(file, path.resolve(DIST_PATH, file.replace(basePath + "/", "./")))
+  }
+}
+
+async function buildRollup() {
+  ensureFileExists(DIST_PATH, "UnityLoader.js")
+  console.log("> reading UnityLoader.js")
+  const banner = readFileSync(path.resolve(DIST_PATH, "UnityLoader.js")).toString()
```

Comment:
```suggestion
  ensureFileExists(DIST_PATH, "DCLUnityLoader.js")
  console.log("> reading DCLUnityLoader.js")
  const banner = readFileSync(path.resolve(DIST_PATH, "DCLUnityLoader.js")).toString()
```

If here we are including the used loader, the correct name would be `DCLUnityLoader.js`.
Note that in the build folder we also have `hls.min.js` for the video streaming plugin. Not sure if that'd break if we don't include it here.